test/parallel/test-http-set-timeout.js

```diff
@@ -1,32 +1,26 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var http = require('http');
-var net = require('net');
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+const net = require('net');
 
 var server = http.createServer(function(req, res) {
-  console.log('got request. setting 1 second timeout');
-  var s = req.connection.setTimeout(500);
-  assert.ok(s instanceof net.Socket);
-  req.connection.on('timeout', function() {
+  console.log('got request. setting 500ms timeout');
+  var socket = req.connection.setTimeout(500);
```

Comment:
Can these vars be `const` as well? It seems a little weird to make them constant, but if they don't change...